doc/source/rllib/index.rst

```diff
@@ -20,20 +20,22 @@ making needs.
 
 You **don't need** to be an **RL expert** to use RLlib, nor do you need to learn Ray or any
 other of its libraries! If you either have your problem coded (in python) as an
-`RL environment <https://medium.com/distributed-computing-with-ray/anatomy-of-a-custom-environment-for-rllib-327157f269e5>`_
+`RL environment <rllib-env.html#configuring-environments>`__
 or own lots of pre-recorded, historic behavioral data to learn from, you will be
 up and running in only a few days.
 
 RLlib is already used in production by industry leaders in many different verticals, such as
 `climate control <https://www.anyscale.com/events/2021/06/23/applying-ray-and-rllib-to-real-life-industrial-use-cases>`_,
-`manufacturing and logistics <https://www.anyscale.com/events/2021/06/22/offline-rl-with-rllib>`_,
+`industrial control <https://www.anyscale.com/events/2021/06/22/offline-rl-with-rllib>`_,
+`manufacturing and logistics <https://www.anyscale.com/events/2022/03/29/alphadow-leveraging-rays-ecosystem-to-train-and-deploy-an-rl-industrial>`_,
 `finance <https://www.anyscale.com/events/2021/06/22/a-24x-speedup-for-reinforcement-learning-with-rllib-+-ray>`_,
 `gaming <https://www.anyscale.com/events/2021/06/22/using-reinforcement-learning-to-optimize-iap-offer-recommendations-in-mobile-games>`_,
 `automobile <https://www.anyscale.com/events/2021/06/23/using-rllib-in-an-enterprise-scale-reinforcement-learning-solution>`_,
 `robotics <https://www.anyscale.com/events/2021/06/23/introducing-amazon-sagemaker-kubeflow-reinforcement-learning-pipelines-for>`_,
 `boat design <https://www.youtube.com/watch?v=cLCK13ryTpw>`_,
 and many others.
 
+`You can read about RLlib Key Concepts. <core-concepts.html>`__
```

Comment:
We shouldn't internally link to HTML; could we instead use a cross-reference? https://docs.readthedocs.io/en/stable/guides/cross-referencing-with-sphinx.html

Reply:
ah, I see.  The rllib github README is just a plain .md file, I need to find the right headers to get it to sphinx-compile.

Reply:
All the other urls on the github rllib README are full urls.  Just replace those new urls with full urls for now, instead of refactoring whole page.